### ISSUE TITLE
Teal primary button fails accessibility contrast check

### ISSUE THREAD

**Issue by hartsick** [codeforamerica/cfa-styleguide-gem#18]
In setting up accessibility specs on Colorado, I ran into a contrast-related failure on primary buttons in teal (`.button--primary`). Currently the buttons look like this (and have a ratio of 3.44):

![screen shot 2018-11-01 at 1 51 34 pm](https://user-images.githubusercontent.com/3675092/47879266-695ccb00-dddd-11e8-8bfc-99713f156953.png)

The middle button is the one failing our contrast tests at the moment, though suspect at least the smaller button is, as well.

Here's the page that describes the failure we experienced, and has some tools to try things out to improve the contrast: https://dequeuniversity.com/rules/axe/2.3/color-contrast?application=axeAPI

**Comment by hartsick** [comment]
All merged in!
